datasets/_externals/ext_cy_companies.yml

```diff
@@ -62,10 +63,8 @@ inputs:
   - debarment
 
 config:
-  type: nomenklatura.enrich.yente:YenteEnricher
-  api: https://api.graph.opensanctions.org/
+  type: zavod.runner.local_enricher:LocalEnricher
```

Comment:
validate

Reply:
ΙΡΙΝΑ SYROKOMSKAIA is missing, and it's fine, because they're not a director any more. cy_companies ran last night, and graph only updates monthly so yente is out of date. Confirmed at https://efiling.drcor.mcit.gov.cy/DrcorPublic/SearchResults.aspx?name=OTKRITIE+CAPITAL+LIMITED&number=%25&searchtype=optStartMatch&index=1&tname=%25&sc=0